ethereum_history_api/circuits/lib/src/verifiers/receipt/split.nr

```diff
@@ -0,0 +1,22 @@
+use dep::proof::{const::MAX_RECEIPT_RLP_LENGTH as MAX_ENCODED_RECEIPT_LENGTH, utils::memcpy};
+use crate::receipt::TxType;
+
+global MAX_RECEIPT_RLP_LENGTH: u64 = MAX_ENCODED_RECEIPT_LENGTH - 1;
+
+pub(crate) fn split_receipt_into_tx_type_and_receipt_rlp(
```

Comment:
imp receipt {
  split_into_tx_and_rlp() -> 

}

Reply:
There is a bug in compiler that blocks me from moving on with this design:
https://www.notion.so/vlayer/Create-a-repro-for-compiling-crashing-on-overlapping-import-aliases-used-in-type-aliases-bad6dde830ff4f40a13726abb58a2a6f?pvs=4
Should I re-request review for this PR without this comment, or switch to reporting the repro and return to this PR when the bug is fixed?